extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -176,6 +177,19 @@ define(
 		$('.editpage-widemode-trigger').click(editorModeChange);
 	}
 
+	function launchEditorPreview(){
+		var editorValue = ace.getContent();
+		var wikiURL = mw.config.get('wgServer').split('://')[1];
+		var templateName = mw.config.get('wgPageName').split(':')[1];
```

Comment:
Can you use wgTitle? Or use mw.Title object to parse wgPageName instead of doing it with split?